samcli/lib/sync/flows/alias_version_sync_flow.py

```diff
@@ -62,19 +63,34 @@ def __init__(
         self._alias_name = alias_name
         self._lambda_client = None
 
+    @property
+    def sync_state_identifier(self) -> str:
+        """
+        Sync state is the unique identifier for each sync flow
+        In sync state toml file we will store
+        Key as AliasVersionSyncFlow:FunctionLogicalId:AliasName
+        Value as alias version number
+        """
+        identifier = self.__class__.__name__ + ":" + self._function_identifier + ":" + self._alias_name
+        return identifier.replace("/", ":")
```

Comment:
Q: Is the replacement of "/" here intended for nested stack names? If so sync_context already replaces "/" with "-" before storing it in the syncstate.

Reply:
I would assume that should be the job of `SyncContext` (or `SyncState`) to keep functionality in one place. Otherwise, we might need to do it in every `SyncFlow` definition.